jpa/boot-read-replica-postgresql/src/main/resources/application.yml

```
@@ -23,4 +25,19 @@ spring:
       ddl-auto: validate
   threads:
     virtual:
-      enabled: true
\ No newline at end of file
+      enabled: true
+
+# spring boot log level property
+logging:
+  level:
+    read-replica-logger : DEBUG
```

yamllint

[warning] too many spaces before colon

(colons)